docs/example-react-hooks-useState.md

```diff
@@ -0,0 +1,81 @@
+---
+id: example-react-hooks-useState
+title: React Hooks useState
+---
+
+## React Hooks useState
+
+This will be a simple example using useState to test a React Hooks functional component.
+
+Here is our component, it is a simple component that changes some text when a user clicks a button: 
+
+```
+import React, { useState } from 'react';
+
+
+const TestHook = (props) => {
```

Comment:
One thing I'd like to avoid is too many examples using "TestXYX" as the name. That makes it sound like this is a special test harness or wrapper for testing. Names like `Example`, `MyComponent`, `App`, or a descriptive name like `ToggleButton` avoid this confusion.